perf_test/blas/blas2/KokkosBlas2_gemv_perf_test.cpp

```diff
@@ -45,6 +45,7 @@
 #include "KokkosBlas2_gemv.hpp"
 #include <Kokkos_Random.hpp>
 #include "KokkosKernels_TestUtils.hpp"
+#include "KokkosKernels_IOUtils.hpp"
```

Comment:
Does the GEMV perf test depend on IOUtils? I tried removing this line and it still compiled.

Reply:
Okay, let me fix that, it will certainly look cleaner without it.